### DISCUSSION TITLE
【灌水】终于进来了

### DISCUSSION THREAD

**Discussion by wenkui921** [share121/inter-knot#3413] [其他]
不过刷新也没有帖子哎（哎我真的进来了吗）
![Screenshot_20240821_174208_com lofter android_edit_473010484272094](https://github.com/user-attachments/assets/19020b6f-9850-48c2-bb4b-4b86656e4963)

**Comment by 3165341164** [comment]
是这样的，因为刚开始阶段再加上难度比较大，所以人少了，帖子也少<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Reply by wenkui921** [reply]
哎？我这边是连帖子都没有哎
![17242539390603295ec0cc3b6e6da05b361435712e17e9d6433ffff480877cda6fbca7ab54f5b 0_edit_493585860726246](https://github.com/user-attachments/assets/5633b03a-2057-486b-a940-58fcda1b450d)
![Screenshot_20240821_190753_mark via_edit_493614529190825](https://github.com/user-attachments/assets/b61e9f88-ceaa-4b5b-bf10-113c2b91811d)

**Comment by Dolor1s** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/bf7e7cae6c42a727faa3d5e76b4a4c1c.png" alt="帕姆_比心" width="75" >

**Comment by Leonie0202** [comment]
你没走错！这里就是绳网！

**Comment by wenkui921** [comment]
求求佬们帮一下
![Screenshot_20240821_190753_mark via_edit_493614529190825](https://github.com/user-attachments/assets/8b643410-69f7-4044-8d1c-0d94ad640b27)
![Screenshot_20240821_232306_mark via_edit_493600167942390](https://github.com/user-attachments/assets/54b4ff76-1b65-4b31-a64c-4b34dd69c9ff)
一直这样看不了怎么办啊
![17242539390603295ec0cc3b6e6da05b361435712e17e9d6433ffff480877cda6fbca7ab54f5b 0_edit_493585860726246](https://github.com/user-attachments/assets/fb57ca96-9677-4d14-8cd4-7f611bb1b742)

**Comment by wenkui921** [comment]
佬们，一直显示获取文章失败该怎么办啊**꒦ິ^꒦ິ**

**Reply by wenkui921** [reply]
看不了帖子(*꒦ິ⌓꒦ີ)

**Reply by wenkui921** [reply]
好像我新文章也发不了了(;´༎ຶД༎ຶ`)

**Reply by 3165341164** [reply]
试一试火狐

**Reply by wenkui921** [reply]
火狐不行，我火狐卡在下脚本那了。
他没有弹出下载脚本的选项

**Comment by Nie7bai** [comment]
emmm……你可能需要一个能下脚本的浏览器（via？）
或者你已经下载了脚本，但仍任然失败，可以清空浏览器的数据，再重新下载脚本

**Reply by wenkui921** [reply]
我就是用via上的qwq
待会试试重新下

**Reply by wenkui921** [reply]
w试了删了重新下但还是不行qwq

**Comment by Leonie0202** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/856d513772f4f83df68fe7854be750d7.png" alt="桑博 满是世俗的欲望" width="75" >为什么会看不了呀？我的话是有开加速器进来的，而且是电脑，如果你加载不了要不要试试去看绳网创始人b站发的视频？会不会是绳网版本和绳网跨域助手的版本不同？

**Reply by wenkui921** [reply]
不知道啊
我这边也有开加速器，不过我是手机，
就是跟着视频下via到注册账号的˃ ˄ ˂̥̥

**Comment by wenkui921** [comment]
QwQ花了一天时间也只进了半个身子꒦ິ^꒦ິ
事已至此，先睡觉吧🛌💤
